bosh/releases/pre_render_scripts/router/gorouter/jobs/patch_pre-start.sh

```diff
@@ -1,30 +0,0 @@
-#!/usr/bin/env bash
-
-set -o errexit -o nounset
-
-target="/var/vcap/all-releases/jobs-src/routing/gorouter/templates/pre-start.erb"
```

Comment:
This one actually gets used, I believe. Given that pre-start scripts run in a _different_ container I'm suspicious of it doing anything useful.  Either way, we will need to check the resulting ulimit… I see that the `gorouter` binary has an actual limit of `1048576` (for both soft and hard), so we're _not_ pulling the value from `bpm.yml`, interesting.

Reply:
Squashed history and amended commit message based on these comments.
Commit 5f7cd28c48738106936b93d1ae215ead9c8d412d